src/rust/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "harfbuzz_rust"
+edition = "2021"
+
+[dependencies]
```

Comment:
Setting a specific 0.x would be nice. I wouldn't have expected you to have to list font-types and read-fonts explicitly.

Reply:
Re setting specific 0.x versions, I'm using harfruzz from git, since 0.0.1 is not on crates.io, and if I set actual numbers for the others, they seem to easily conflict with what harfruzz requests?

Reply:
https://github.com/harfbuzz/harfruzz/pull/19 to setup to publish harfruzz and be able to distinguish breaking vs non-breaking changes

Reply:
Thanks. For now I pinned to a git hash.

Reply:
https://crates.io/crates/harfruzz now has a 0.1.0